src/CloudStructures/IRedisConnection.cs

```diff
@@ -0,0 +1,25 @@
+using StackExchange.Redis;
+
+namespace CloudStructures
+{
+    /// <summary>
+    /// Represents a Redis connection interface.
+    /// </summary>
+    public interface IRedisConnection
```

Comment:
where does this get used?

Reply:
The RedisConnection class implements this interface